huxley/www/static/js/huxley/components/RegistrationView.js

```diff
@@ -89,6 +89,14 @@ var RegistrationView = React.createClass({
     };
   },
 
+  componentWillMount: function() {
+    if (!conference.registration_open) {
```

Comment:
@tdowds Yeah so if a user navigates to "/register" not using the UI then they will be able to see the registration page which is something we don't want. 

@kmeht Initially didn't implement it like that because of the file structure but I will take your suggestion as it is cleaner